README.md

```diff
@@ -20,6 +20,66 @@ Fork this repository and add your tables here:
 
 (your tables go here - you can use TableGenerator to create them. Include the data from the sample above in your tables)
 
+### Authors
+
+| AuthorID | FullName      | Email                       | Bio            |
+|----------|---------------|-----------------------------|----------------|
+| 1        | Eric Fleming  | ericfleming@nimblepros.com  | Long Text Here |
+| 2        | Kyle McMaster | kylemcmaster@nimblepros.com | Long Text Here |
+| 3        | Steve Smith   | steve@nimblepros.com        | Long Text Here |
+
+### BlogPosts
+
+| Slug                          | Title                                | Body           | AuthorID | Views |
+|-------------------------------|--------------------------------------|----------------|----------|-------|
+| serilog-structured-logging    | Structured Logging with Serilog      | Long Text Here | 1        | 123   |
+| persisting-smart-enum         | Persisting a Smart Enum with EF Core | Long Text Here | 2        | 234   |
+| localization-aspnet-core-apis | Localization in ASP.NET Core APIs    | Long Text Here | 3        | 345   |
+
+### Categories
+
+| CategoryID | Category             |
+|------------|----------------------|
+| 1          | Software Development |
+| 2          | Cloud Computing      |
+| 3          | Web APIs             |
+
+### Tags
+
+| TagID | Tag          |
+|-------|--------------|
+| 1     | logging      |
+| 2     | serilog      |
+| 3     | appinsights  |
+| 4     | smartenum    |
+| 5     | efcore       |
+| 6     | localization |
+| 7     | aspnetcore   |
+| 8     | web-api      |
```

Comment:
Looks good. Now, how would you modify this (if at all) if your users wanted to use any case they prefer for their tags, and they wanted that case to appear on the page as they entered it. So you might have `aspnetcore` as a tag for one article but `AspNetCore` as a tag for another article.

But when a user searches for all articles tagged `aspnetcore` it should return both of the above articles.

Thoughts?

Reply:
You can set the case sensitivity collation at server level when you create the table / database or change it later, or more simple and thus requiring no modification is to use a COLLATE clause when doing a query.

Reply:
Right so store them uniquely (each case variation), but retrieve them regardless of case. Sounds good.